tools/logicalimager/LogicalImagerRuleSet.cpp

```diff
@@ -53,8 +53,8 @@ time_t stringToTimet(const std::string &datetimeStr) {
     return time;
 }
 
-int getPositiveInt(const std::string &key, nlohmann::json ruleJson) {
-    int size;
+TSK_OFF_T getPositiveInt(const std::string &key, nlohmann::json ruleJson) {
```

Comment:
The date rule also calls this and is now generating a warning about the conversion from TSK_OFF_T to int